pkg/stanza/operator/input/windows/input.go

```diff
@@ -42,6 +82,11 @@ func (i *Input) Start(persister operator.Persister) error {
 
 	i.persister = persister
 
+	if err := i.startRemoteSession(); err != nil {
+		i.Logger().Error("Failed to start remote session", zap.Error(err))
+		return err
```

Comment:
Returning an error from `Start` will take down the collector. We typically try to avoid this for situations where the remote server may be temporarily unavailable. I think we should just log the error here and check for a session during each interval.

Reply:
updated!

Reply:
Per documentation `EvtOpenSession` doesn't validate credentials. @JonathanWamsley could you please check what it returns if the targeted machine doesn't exist or is offline?